src/schema/fair.ts

```diff
@@ -47,6 +49,44 @@ const FairType = new GraphQLObjectType({
   name: "Fair",
   fields: () => ({
     ...GravityIDFields,
+    artistsConnection: {
+      type: artistConnection,
+      args: pageable({
+        section: {
+          type: GraphQLString,
+          description: "Number of artists to return",
```

Comment:
I'll take your word for it, but just double-checking that this is what "section" means?

Reply:
Nothing apparently! I copied it over from a similar connection and had assumed it was an internal pagination parameter, but things appear to work fine without it - thanks for calling it out.